module/evdi_modeset.c

```diff
@@ -26,6 +26,9 @@
 #include "evdi_cursor.h"
 #include "evdi_params.h"
 #include <drm/drm_gem_framebuffer_helper.h>
+#if KERNEL_VERSION(5, 0, 0) <= LINUX_VERSION_CODE
```

Comment:
It looks like EL8 has this support present.  You should add a check for it:

`#if KERNEL_VERSION(5, 0, 0) <= LINUX_VERSION_CODE || defined(EL8)`